src/components/Results.js

```diff
@@ -26,7 +26,7 @@ const Results = (props) => {
 
     if (gameWinner.evoSprites) {
         return (
-            <section className="results">
+            <>
```

Comment:
I think this fragment is unnecessary.

Reply:
We're using it to contain everything within the return -- the section with the results class was moved one level up to DisplayGame to also contain the play again button within it

Reply:
sorry, the diff in github made it look like there was only one div instead of two :)